cmd/dagger/module.go

```diff
@@ -117,6 +118,9 @@ func moduleAddFlags(cmd *cobra.Command, flags *pflag.FlagSet, optional bool) {
 		defaultAllowLLM = strings.Split(allowLLMEnv, ",")
 	}
 	flags.StringSliceVar(&allowedLLMModules, "allow-llm", defaultAllowLLM, "List of URLs of remote modules allowed to access LLM APIs, or 'all' to bypass restrictions for the entire session")
+
+	// Add the eager module loading flag to disable lazy load on runtime.
+	flags.BoolVar(&eagerModuleLoading, "eager-mod-loading", false, "load module runtime eagerly")
```

Comment:
Why not something a bit simpler, like `--load-runtime`? Especially if the goal is to load the runtime.